schemas/2016-06-01/Microsoft.RecoveryServices.json

```diff
@@ -85,40 +82,25 @@
             {
               "type": "string",
               "enum": [
+                "Standard",
```

Comment:
We don't support name as Standard now. Earlier BackupVault used to support Standard as name, which shouldn't be used now.